app/src/engineering/kotlin/com/hedvig/app/feature/embark/EmbarkStoryTesterActivity.kt

```diff
@@ -253,33 +253,22 @@ class EmbarkStoryTesterViewModel(
   }
 
   fun setAuthToken(authToken: String) {
-    authenticationTokenService.authenticationToken = authToken
+    authTokenService.updateTokens(AccessToken(authToken, 600), RefreshToken(authToken, 600))
```

Comment:
Not sure this'd work when testing since we'd not have a proper refresh token. I know you've built and used the EmbarkStoryTesterActivity before, maybe this will need some touch up before it's gonna be usable again.

Reply:
No worries - dont think we will use this activity much now that we are working mostly with odyssey.